packages/db/postgres/tables.sql

```diff
@@ -247,3 +246,12 @@ CREATE TABLE IF NOT EXISTS name
     inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
     updated_at  TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS pinning_authorization
+(
+  id              BIGSERIAL PRIMARY KEY,
+  -- Points to user allowed to pin content.
+  user_id     BIGINT                                                       NOT NULL REFERENCES public.user (id),
+  inserted_at     TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
+  deleted_at      TIMESTAMP WITH TIME ZONE
```

Comment:
Some weird spacing here!

```suggestion
  id          BIGSERIAL PRIMARY KEY,
  -- Points to user allowed to pin content.
  user_id     BIGINT NOT NULL REFERENCES public.user (id),
  inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
  deleted_at  TIMESTAMP WITH TIME ZONE
```

Reply:
Fixed.